src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -543,10 +543,14 @@ well as menu structures (for main menu and popup menus).
       <!-- Separate shortcuts with '|', e.g. 'Ctrl+X Ctrl+F|Cmd+O' -->
       <shortcutgroup name="Source Editor">
          <shortcut refid="insertChunk" value="Cmd+Alt+I"/>
-         <shortcut refid="insertSection" value="Cmd+Shift+R"/>
+         <shortcut refid="insertSection" value="Cmd+Shift+R" disableModes="sublime"/>
+         <shortcut refid="insertSection" value="Ctrl+Shift+R" if="org.rstudio.core.client.BrowseCap.isMacintosh()"/>
```

Comment:
Does <kbd>Cmd + Shift + R</kbd> still work for inserting a section on macOS when Sublime keybindings are disabled?

Reply:
Sorry, I'm guessing it should work given the command binding immediately above.